proto-google-cloud-aiplatform-v1beta1/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,338 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<!-- TODO: remove after next major version is released -->
+<differences>
+  <difference>
+    <className>
+      com/google/cloud/aiplatform/v1beta1/BatchMigrateResourcesOperationMetadataOrBuilder
+    </className>
+    <differenceType>7012</differenceType>
+    <method>* getPartialResults(*)</method>
+  </difference>
+
+  <difference>
+    <className>
+      com/google/cloud/aiplatform/v1beta1/BatchMigrateResourcesOperationMetadataOrBuilder
+    </className>
+    <differenceType>7012</differenceType>
+    <method>* getPartialResultsCount()</method>
+  </difference>
```

Comment:
Generally, for the protos, we use a wildcard matcher for these. Example: https://github.com/googleapis/java-asset/blob/911e403a2d149f24a40bc7c99021777542ab6a8a/proto-google-cloud-asset-v1/clirr-ignored-differences.xml#L4-L18